noetic/distribution.yaml

```diff
@@ -1675,6 +1675,21 @@ repositories:
       url: https://github.com/bostoncleek/ergodic_exploration.git
       version: noetic-devel
     status: developed
+  ethercat_grant:
+    doc:
+      type: git
+      url: https://github.com/shadow-robot/ethercat_grant.git
+      version: noetic-devel
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/shadow-robot/ethercat_grant-release.git
+      version: 0.2.5-1
+    source:
+      type: git
+      url: https://github.com/shadow-robot/ethercat_grant.git
```

Comment:
Please add a top-level LICENSE file to the source repository (per our [review guidelines](https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md)). No need to make a new release.

Reply:
I see that you've added a [LICENSE file](https://github.com/shadow-robot/ethercat_grant/blob/noetic-devel/LICENSE.md) (thanks!). However, the new file claims the code is GPL, but the package.xml lists BSD. Please clarify how the code is licensed by updating either the LICENSE file or package.xml.

Reply:
Verified, by mistake put GPL. License is BSD and it was updated to be like in package.xml.

Reply:
Thanks for iterating!